bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html

```diff
@@ -0,0 +1,112 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+
+{% add_lang_files "firefox/whatsnew_76" %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox - Make Firefox your default browser') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_76') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew-76' %}
+{% set _utm_campaign = 'whatsnew-76' %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success up-to-date">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-c-emphasis-box">
```

Comment:
Nit: this emphasis box should probably be inside a `<div class="mzp-l-content"></div>` container?. Without it it runs flush to the viewport at smaller screen sizes.